cookbook/common/leaf.mk

```diff
@@ -66,101 +80,42 @@ requirements.txt: requirements.in install-piptools
 
 .PHONY: requirements
 requirements: requirements.txt
+	@echo ${CURDIR}
+	pip install -r requirements.txt
 
-.PHONY: fast_serialize
-fast_serialize: clean _pb_output
-	echo ${CURDIR}
-	docker run -it --rm \
-		-e REGISTRY=${REGISTRY} \
-		-e MAKEFLAGS=${MAKEFLAGS} \
-		-e FLYTE_HOST=${FLYTE_HOST} \
-		-e INSECURE_FLAG=${INSECURE_FLAG} \
-		-e PROJECT=${PROJECT} \
-		-e FLYTE_AWS_ENDPOINT=${FLYTE_AWS_ENDPOINT} \
-		-e FLYTE_AWS_ACCESS_KEY_ID=${FLYTE_AWS_ACCESS_KEY_ID} \
-		-e FLYTE_AWS_SECRET_ACCESS_KEY=${FLYTE_AWS_SECRET_ACCESS_KEY} \
-		-e OUTPUT_DATA_PREFIX=${OUTPUT_DATA_PREFIX} \
-		-e ADDL_DISTRIBUTION_DIR=${ADDL_DISTRIBUTION_DIR} \
-		-e SERVICE_ACCOUNT=$(SERVICE_ACCOUNT) \
-		-e VERSION=${VERSION} \
-		-v ${CURDIR}/_pb_output:/tmp/output \
-		-v ${CURDIR}:/root/$(shell basename $(CURDIR)) \
-		${TAGGED_IMAGE} make fast_serialize
+.PHONY: serialize
+serialize: docker_build
+	@echo ${VERSION}
+	pyflyte --pkgs ${PREFIX}  package --image ${TAGGED_IMAGE} --source=${CURDIR}/..  --force --output="${OUTPUT_DIR}flytesnacks_${PREFIX}.tgz"
 
-.PHONY: fast_register
-fast_register: clean _pb_output ## Packages code and registers without building docker images.
-	@echo "Tagged Image: "
-	@echo ${TAGGED_IMAGE}
-	@echo ${CURDIR}
-	docker run -it --rm \
-		--network host \
-		-e REGISTRY=${REGISTRY} \
-		-e MAKEFLAGS=${MAKEFLAGS} \
-		-e FLYTE_HOST=${FLYTE_HOST} \
-		-e INSECURE_FLAG=${INSECURE_FLAG} \
-		-e PROJECT=${PROJECT} \
-		-e FLYTE_AWS_ENDPOINT=${FLYTE_AWS_ENDPOINT} \
-		-e FLYTE_AWS_ACCESS_KEY_ID=${FLYTE_AWS_ACCESS_KEY_ID} \
-		-e FLYTE_AWS_SECRET_ACCESS_KEY=${FLYTE_AWS_SECRET_ACCESS_KEY} \
-		-e OUTPUT_DATA_PREFIX=${OUTPUT_DATA_PREFIX} \
-		-e ADDL_DISTRIBUTION_DIR=${ADDL_DISTRIBUTION_DIR} \
-		-e SERVICE_ACCOUNT=$(SERVICE_ACCOUNT) \
-		-e VERSION=${VERSION} \
-		-v ${CURDIR}/_pb_output:/tmp/output \
-		-v ${CURDIR}:/root/$(shell basename $(CURDIR)) \
-		${TAGGED_IMAGE} make fast_register
+.PHONY: fast_serialize
+fast_serialize:
+	@echo pyflyte --pkgs ${PREFIX} package --image ${TAGGED_IMAGE} --source=${CURDIR}/..  --force --fast --output="${OUTPUT_DIR}flytesnacks_${PREFIX}.tgz"
```

Comment:
I think we need to add `--in-container-source-path` here too...

Reply:
no, it's optional...I think it is removed in the pyflyte package